tests/Unit/ObfuscationTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
🔝

Reply:
Not sure I understand this comment. Hehe.

Reply:
eheheh nothing specific, I love when we have specific classes/modules that can be independently unit tested 😄 so it was just excitement 😄